packages/realm_dart/lib/src/cli/atlas_apps/baas_client.dart

```diff
@@ -321,15 +391,18 @@ class BaasClient {
     await _updateFunction(app, 'confirmFunc', confirmFuncId, source ?? _confirmFuncSource);
   }
 
-  Future<BaasApp> _createApp(String name, String suffix, {String? confirmationType}) async {
-    final uniqueName = "$name$suffix";
+  Future<BaasApp> _createApp(AppNames appName) async {
+    final uniqueName = "${appName.name}$_appSuffix";
     print('Creating app $uniqueName');
 
+    final runConfirmationFunction = appName != AppNames.autoConfirm && appName != AppNames.emailConfirm;
```

Comment:
Looks odd here due to plural
```suggestion
    final runConfirmationFunction = appName != AppName.autoConfirm && appName != AppName.emailConfirm;
```